domains/vitaliy.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "TkachenkoVitaliy",
+        "email": "tkachenko.java@yandex.ru"
+    },
+    "record": {
+        "A": ["45.155.207.175"]
```

Comment:
I cannot reach your website ("45.155.207.175 refused to connect.")

Reply:
Sorry, the server hosting the website went down, but it's active now.